opencti-platform/opencti-front/src/private/components/settings/groups/Group.tsx

```diff
@@ -344,6 +407,64 @@ const Group = ({ groupData }: { groupData: Group_group$key }) => {
               </Grid>
             </Paper>
           </Grid>
+          <Grid item={true} xs={12} style={{ marginTop: 30 }}>
+            <Typography
+              variant="h4"
+              gutterBottom={true}
+              style={{ float: 'left' }}
+            >
+              {t('Triggers and Digests')}
```

Comment:
no upper case for the first letter of 'digest' in the translations

Reply:
fixed